GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -225,16 +225,42 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
+                //// MenuCommands not disposable
+                //// _superprojectCurrentCheckout not disposable
+                //// _selectedObjectIds not disposable
+                //// _baseCommitToCompare not disposable
                 _revisionSubscription?.Dispose();
                 _revisionReader?.Dispose();
+                //// _artificialStatus not disposable
+                //// _ambiguousRefs not disposable
+                //// _refFilterOptions not disposable
+                _lastVisibleResizableColumn?.Dispose();
+                _maximizedColumn?.Dispose();
+                //// _revisionGraphColumnProvider not disposable
+                _selectionTimer?.Dispose();
```

Comment:
❕ `_selectionTimer` is added to the components collection when it is constructed. It will be disposed automatically as part of that collection and should not be separately disposed here.